ivadomed/utils.py

```diff
@@ -536,7 +554,13 @@ def segment_volume(folder_model, fname_image, gpu_number=0, options=None):
     for i_batch, batch in enumerate(data_loader):
         with torch.no_grad():
             img = cuda(batch['input'], cuda_available=cuda_available)
-            preds = model(img) if fname_model.endswith('.pt') else onnx_inference(fname_model, img)
+
+            if ('FiLMedUnet' in context and context['FiLMedUnet']['applied']) or \
+                    ('HeMISUnet' in context and context['HeMISUnet']['applied']):
+                metadata = imed_training.get_metadata(batch["input_metadata"], model_params)
+                preds = model(img, metadata)
```

Comment:
No need to `onnx` because we are always going with `.pt` for FiLM models? [here](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/training.py#L340)

Reply:
Yes currently there is an issue: FiLM models can't be saved in onnx format so it can only be saved in .pt  see https://github.com/ivadomed/ivadomed/issues/462